### ISSUE TITLE
QOL: Scale reach range with size

### ISSUE THREAD

**Issue by Luingar** [naelstrof/KoboldKare#49]
When very large, it is difficult to reach things, pretty much at all, since your height can exceed your reach.

**Comment by naelstrof** [comment]
Yeah good idea. I got worried that large characters would just suck up the whole world of items, but making the grab longer rather than bigger should address that just fine.
